### DISCUSSION TITLE
An error occured while initializing the dbt project: No module named 'dbt'

### DISCUSSION THREAD

**Discussion by bhaskoro-muthohar** [AltimateAI/vscode-dbt-power-user#406] [General]
### Expected behavior

power-dbt success to initiating

### Actual behavior

![CleanShot 2023-04-17 at 14 40 56@2x](https://user-images.githubusercontent.com/35159954/232417907-813c0322-1e9c-42c9-9259-ec0f47981200.png)
power-dbt failed to initializing

### Steps To Reproduce

1. Install power-dbt plugin
2. Restart vscode

### Log output/Screenshots

<img width="451" alt="CleanShot 2023-04-17 at 14 43 07@2x" src="https://user-images.githubusercontent.com/35159954/232418359-44ac8944-2cb7-4614-a606-ae4232d6360c.png">

### Operating System

ProductName: macOS		ProductVersion: 13.0		BuildVersion: 22A380

### dbt version

Core:   - installed: 1.2.5

### dbt Adapter

Plugins: - bigquery: 1.2.0

### dbt Power User version

v0.12.2

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by mdesmet** [comment]
How is dbt installed? (globally, venv?)? Did you point to the right environment?

**Reply by slvaguilar** [reply]
Hi @mdesmet , I am facing the same issue with dbt core 1.8.2 and dbt power user 0.39.20 and the Snowflake 1.8.3 adapter. I have dbt both globally and in a venv. Is this something you can help me with? Thanks.

**Comment by mdesmet** [comment]
See also https://github.com/innoverio/vscode-dbt-power-user/issues/87
